app/controllers/state_file/notifications_settings_controller.rb

```diff
@@ -34,7 +34,7 @@ def subscribe_email
     def matching_intakes(email_address)
       return if email_address.blank?
 
-      StateFileAzIntake.where(email_address: email_address) + StateFileNyIntake.where(email_address: email_address)
+      StateFile::StateInformationService.state_intake_classes.map { |klass| klass.where(email_address: email_address) }.inject([], :+)
```

Comment:
i guess i could also do a reduce here, open to suggestions

Reply:
I think this is actually more concise and clearer than a reduce would be:

`.inject([]) { |result, klass| result.append(klass.where(email_address: email_address)) }`